### ISSUE TITLE
[FEATURE SUGGESTION] Add "done - no further work" message to "Operation in Progress" log message

### ISSUE THREAD

**Issue by david-bakin** [BiglySoftware/BiglyBT#1814]
[Windows Server 2016, BiglyBT Beta 2.5.0.1_B10]

I frequently get torrents (via hash) that end up like this in the "operation in progress window":

```
Downloading From:magnet:?xt=urn:btih:7BRVQ4PF2E635OCFKEZGF....CJOU
Searching...
Metadata download created
```

They just end there, after `metadata download created`.  It's been happening more often the last week or so, don't know if that's related to the torrents I'm downloading or recent beta versions.

Anyway, the feature suggestion is to add a "done" message at the end of the message log when there's no more work being done for that torrent file download.  For example, because all available peers that could have had the torrent file have timed out, or maybe because it couldn't even be found in the DHT?  (Whatever, I don't know all the reasons.)  Anyway I run with timeout=0 so the windows just stay there and do nothing.  Except - this I don't understand - every once in awhile after about 10 minutes or more one of these pending windows wakes up and actually does something like download a torrent.  

So basically I don't understand the "lifecycle" of the "operation in progress" dialog and feel that a "done" message would help.  Or maybe you have a different idea.

And, one more thing, since there's no forum for BiglyBT and no other way to sound off with approval feedback.  Recently a change was made so that if you change the download directory name in the torrent download dialog (the `Subfolder` edit control) then the torrent display name in the incomplete torrent grid changes to match ... bravo!!! Thank you! It was a constant nuisance that that didn't happen before ... glad that's fixed.

**Comment by parg** [comment]
If you have a timeout value of 0 (infinite) then the metadata download will run forever trying to find some peer to grab the metadata from - if there are no peers then nothing will happen, as you know :) If there was a timeout then it would eventually say something like 'failed, no sources for torrent'.

The metadata download periodically looks for peers in the DHT, I suppose I could add a periodic 'still trying!' message to the window but really that would just fill it up. The reason it doesn't say 'done' is because, with an infinite timeout, it never is (unless it happens to actually grab the metadata)
